app/lib/gpDetailsParser.js

```diff
@@ -1,6 +1,8 @@
 const xml2js = require('xml2js');
 const jsonQuery = require('json-query');
 const stripPrefix = require('xml2js/lib/processors').stripPrefix;
+const assert = require('assert');
+const Verror = require('verror');
```

Comment:
The casing can be whatever we want - they also use var and " in their code samples. 
I went for simple capitalised class name.